src/Symfony/Component/Notifier/Channel/ChatChannel.php

```diff
@@ -26,10 +25,6 @@ class ChatChannel extends AbstractChannel
 {
     public function notify(Notification $notification, Recipient $recipient, string $transportName = null): void
     {
-        if (null === $transportName) {
-            throw new LogicException('A Chat notification must have a transport defined.');
-        }
```

Comment:
I don't see a reason for this restriction. All the other channels don't have it either.